sw/host/cryptotest/testvectors/parsers/wycheproof_ecdh_parser.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import json
+import jsonschema
+import logging
+import sys
+
+from Crypto.PublicKey import ECC
+
+# Mapping from the curve names used by Wycheproof to those used by cryptotest
+EC_NAME_MAPPING = {
+    "secp256r1": "p256",
+    "secp384r1": "p384",
+}
+
+
+def parse_test_vectors(raw_data):
+    test_groups = raw_data["testGroups"]
+    test_vectors = list()
+    for group in test_groups:
+        # Parse tests within the group
+        for test in group["tests"]:
+            logging.debug(f"Parsing tcId {test['tcId']}")
+            # Reject tests with invalid ASN
```

Comment:
For future users, can you elaborate a bit more in this comment about why we are rejecting InvalidASN-tagged tests?